docs/stability.rst

```diff
@@ -134,11 +134,10 @@ The current planned and existing sub-packages are:
                 astropy.cosmology
             </td>
             <td align='center'>
-                <span class="stable"></span>
+                <span class="mature"></span>
             </td>
             <td>
-                Incremental improvements since v0.1, but mostly stable API.
-                Pure functional interface deprecated in v0.4.
+                Incremental improvements since v0.1, stable API last several interfaces.
```

Comment:
Don't understand "stable API last several interfaces"?

Reply:
typo, was supposed to be "versions"